boot/LoadLinux.c

```diff
@@ -188,7 +189,7 @@ CONFIGENTRY *DetectLinuxFATX(FATXPartition *partition) {
 	return config;
 }
 
-int LoadLinuxFATX(FATXPartition *partition, const OPTLINUX *optLinux) {
+int LoadLinuxFATX(FATXPartition *partition, OPTLINUX *optLinux) {
```

Comment:
Why drop the `const`? The new prototype in `loaders.h` has the `const`

Reply:
The `const` is dropped because the FATX functions, which receive `optLinux->szKernel` as a parameter, are modifying the filename parameter they get, so keeping this const would violate const correctness.
The mismatch with `loaders.h` is a copy&paste error, I created that header before I modified this function. I added `#include "loaders.h"` to the `*Load.c` files so that this mismatch can't happen anymore.

The better solution would be to not let the FATX functions modify their string parameters, but that requires deeper modifications and I would prefer if that happened in another PR.